app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+class CommentsController < ApplicationController
+	def create
+		@postquote = Postquote.find_by(id: params[:postquote_id])
+		create_comment(@postquote)
+	end
+
+	private
+	def create_comment(postquote)
+		@comment = @postquote.comments.create(comment_params)
+		if @comment.save
+			flash[:notice] = "Thank you for submitting a comment"
+			
+		else
+			flash[:alert] = "Sorry, your comment failed to save, please try again"
+		end
+		redirect_to postquote_path(@postquote.id)
```

Comment:
Think about how you might re-render the form the user submitted the quote from. If you redirect after a failure, the user will lose the data they entered.